shimmer/learn/welcome.md

```diff
@@ -0,0 +1,50 @@
+---
+id: welcome
+title: Welcome to Shimmer
+description: Shimmer is a distributed ledger technology with advanced tokenization, smart contracts, and a unique data structure.
+keywords:
+  - explanation
+  - shimmer
+  - shimmie
+image: /img/iota-wiki.png
+---
+
+# Welcome to Shimmer
+
+Shimmer is a decentralized ledger with the support of NFTs, native tokens, and smart contracts. This is an innovation network where the community tries out the new protocol changes before deploying them on the more conservative [IOTA network](https://wiki.iota.org). It has its own supply of tokens which are called $SMR.
+
+## Basics
+
+### Nodes
+
+Shimmer is a network made out of *nodes*, computers that run the Shimmer node software. Anyone can join and leave the network, and nodes are run by individual parties. If you own an average computer, you can run your own Shimmer node, too.
+
+### Blocks and Outputs
+
+The structure that stores the data is called the [Tangle](./tangle.md). It consists of separate *blocks*, each of which stores some small data change. By including blocks into the Tangle, nodes agree on how the Tangle looks like. To get included, each block should reference from two to eight older blocks, and be referenced by a newer block. It means that there is always a front of the most recent blocks: you even can [see](https://explorer.shimmer.network/testnet/visualizer/) it. The more new pending blocks there are, the wider the front and the more points are there for newer blocks to attach. This scales really well. 
+
+Now, when the nodes agreed on the Tangle state, they can extract useful information from it: for example, who owns what tokens. Each block specifies outputs. In the most basic form, outputs specify an amount of tokens and their owner. Blocks exhaust existing outputs and create new outputs with the same total amount of tokens, but with different distribution and owners. Each output can be spent only once: all future blocks that try to exhaust it will be rejected by the nodes. You can ask a node to go through all blocks for a list of created and exhausted outputs that mention you as an owner. If you strike out all exhausted outputs from the list of created outputs, you will get a list of *unspent outputs*. The sum of their tokens will make your token balance.
```

Comment:
```suggestion
Now, when the nodes agreed on the Tangle state, they can extract useful information from it: for example, who owns what tokens. According to the UTXO model, blocks specify outputs. In the most basic form, outputs specify an amount of tokens and their owner. Blocks exhaust existing outputs and create new outputs with the same total amount of tokens, but with different distribution and owners. Each output can be spent only once: all future blocks that try to exhaust it will be rejected by the nodes. You can ask a node to go through all blocks for a list of created and exhausted outputs that mention you as an owner. If you strike out all exhausted outputs from the list of created outputs, you will get a list of *unspent outputs*. The sum of their tokens will make your token balance.
```